dev/dll/SharedHelpers.cpp

```diff
@@ -257,30 +257,38 @@ bool SharedHelpers::IsAPIContractV3Available()
 
 void* __stdcall winrt_get_activation_factory(std::wstring_view const& name);
 
-bool SharedHelpers::IsInFrameworkPackage()
+bool IsInPackage(std::wstring_view detectorName)
 {
-    static bool isInFrameworkPackage = []() {
```

Comment:
Why remove the static result caching?

Reply:
We still have them in IsInCBSPackage and IsInFramePackage.

I made IsInFrameworkPackage an shared function and renamed it to IsInPackage, so IsInPackage doesn't need the static.

Reply:
Ah. Yeah, see that now.

---
In reply to: [516225118](https://github.com/microsoft/microsoft-ui-xaml/pull/3520#discussion_r516225118) [](ancestors = 516225118)